hadoop-yarn-project/hadoop-yarn/hadoop-yarn-ui/README.md

```diff
@@ -62,3 +67,21 @@ YARN UI has replaced NPM with Yarn package manager. And hence Yarn would be used
 ### Adding new routes (pages), controllers, components etc.
 
 * Use ember-cli blueprint generator - [Ember CLI](http://ember-cli.com/extending/#generators-and-blueprints)
+
+### Building with Maven
+
+[YARN-6278](https://issues.apache.org/jira/browse/YARN-6278)
+added build profile using frontend-maven-plugin which
+automatically installs Node.js and Yarn locally under target/webapp directory.
+After building yarn-ui by `mvn package -Pyarn-ui`, you can reuse
+locally installed Node.js and Yarn instead of manually installing them.
```

Comment:
"you can reuse the locally installed..."

Reply:
fixed.